### ISSUE TITLE
[Feature Request] Display which mods that overwrite vanilla files

### ISSUE THREAD

**Issue by EnderShadow** [WojciechKrysiak/SCModManager#22]
As the issue title suggests, a tab that lists which mods overwrite vanilla files would be nice. That would make it easier to determine which mods may or may not work in the latest version of the game if they aren't updated.

**Comment by WojciechKrysiak** [comment]
In theory it would be possible, though I wasn't able to accurately locate the Stellaris directory through any of the open Steam/other API's.

It would require the user to provide a path to the executable, but this would also be helpful in starting the game from the manager itself.

**Comment by EnderShadow** [comment]
Starting the game directly through the exe isn't a good idea. If you do that it doesn't connect to the steam workshop and so any mods from the steam workshop automatically get disabled. Unless you figure out how steam launches the game, your best bet is to use the same url that a shortcut would use to launch the game through steam.

**Comment by ThibautSF** [comment]
Yes the best is to use [Steam Browser Protocol](https://developer.valvesoftware.com/wiki/Steam_browser_protocol).

[steam://run/281990](steam://run/281990) for Stellaris.
